components/search/filter-sidebar.js

```diff
@@ -73,6 +75,45 @@ class FilterSidebar extends React.Component {
 
   onChangeFilter (filterName) {
     return ((e) => {
+      // Input Validations
+      switch(filterName) {
+      case 'asnFilter':
+        var asnValue = e.target.value
+        var asnRegEx = /^(^AS|as)?[0-9]+$/
+        if (asnValue && asnValue.match(asnRegEx) === null) {
+          this.setState({
+            asnError: 'Valid formats: AS1234, 1234',
+            [filterName]: asnValue,
+            isFilterDirty: false
+          })
+          return
+        } else {
+          this.setState({
+            asnError: false
+          })
+        }
+        break
+
+      case 'inputFilter':
+        var inputValue = e.target.value
+        // Regex from: https://www.regexpal.com/93652
+        // eslint-disable-next-line no-useless-escape
+        var inputRegex = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/
```

Comment:
As discussed, we should change this to use a domain name and only check if it's a valid domain name